packages/@okta/courage-for-signin-widget/webpack.config.js

```diff
@@ -117,7 +118,12 @@ const webpackConfig = {
         to: `${I18N_DIR}/dist/properties/`,
       }
     ]),
-
+    new ReplacePlugin({
+      patterns: [{
+        regex: /underscore\.noConflict\(\)/,
```

Comment:
Should this be `underscore.default.noConflict()` => `underscore.default` ?

Reply:
I was assuming 
- the replacement plugin takes effect before babel compilation
- therefore, the dist file will be updated

Reply:
@haishengwu-okta Would it be better to update underscore in courage only, and let it own that dependency? SIW just get its copy of `_` from courage. Then the only change here in SIW would be just updating to latest courage.

Reply:
I believe that's the goal! cc @jmelberg-okta 
It would take extra steps (upgrade underscore in courage, make sure okta-ui and okta-core are both green)

Reply:
There is cycle dependency to solve if we want to get rid of underscore from `widget/package.json`
(`util/Bundles` can not depend on `okta`)

```
okta -> util/Bundles -> underscore
        -> underscore
```

Reply:
Courage's dependency on `underscore` unfortunately comes with a myriad of test failures once we upgrade from `1.12.0` to `1.12.1` (changes do `_.debounce`). Upgrading is our end goal, but we'll need some extra time to address the failures.

Reply:
Bumping @aarongranick-okta's [comment](https://github.com/okta/okta-signin-widget/pull/1812/files#r608128581), this should be:

```suggestion
        regex: /underscore.default.noConflict\(\)/,
```

And the replacement value is: `underscore.default`;